### DISCUSSION TITLE
Welcome to YABS Discussions!

### DISCUSSION THREAD

**Discussion by masonr** [masonr/yet-another-bench-script#34] [Announcements]
## 👋 Welcome to YABS Discussion Board!
  This Discussion section will act as a place to connect with other YABS powerusers and benchmark aficionados. Some example discussion topics might be:
  * Ask questions on how to interpret benchmark results
  * Share amazing YABS results!
  * Asking for help when encountering issues running the script
  * Engage with YABS users
  * Propose enhancements that could be made to the YABS script
  * Anything else related to YABS

Thanks for checking out YABS and I hope you enjoy the tool!

**Comment by dpc302** [comment]
Hey- is there a way to add additional iperf servers? I've got about ~16 total id like to test with and would love to find a way to do so. As well as add some of my own hosted 10/20/40g test links on my servers.
Additionally is it possible to view the direct iperf links currently in the script?

Thank you!

**Reply by masonr** [reply]
> Hey- is there a way to add additional iperf servers? I've got about ~16 total id like to test with and would love to find a way to do so. As well as add some of my own hosted 10/20/40g test links on my servers. Additionally is it possible to view the direct iperf links currently in the script?
> 
> Thank you!

Hey there! Yes you can add your own iperf servers if you like. You will need to download the script locally (`curl -L yabs.sh > yabs.sh`) and then modify the iperf server array. As of v2022-06-11, the "IPERF_LOCS" array is on line 680.

The entries can be added one per line using the example, which has 5 elements per entry:

> "lon.speedtest.clouvider.net" "5200-5209" "Clouvider" "London, UK (10G)" "IPv4|IPv6" \

1. host name or IP address of iperf3 server
2. available ports that the server is listening on
3. name of provider 
4. location of servers and link capacity
5. IPv4 for v4 only, IPv6 for v6 only, IPv4|IPv6 for both

Then to run a new test with your modified yabs, run just that file (`bash yabs.sh`). Add "-dg" to the end if you just want to run the iperf tests and nothing else. Give me a shout if you have any questions.
